src/System.Console/tests/RedirectedStream.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class RedirectedStream : RemoteExecutorTestBase
+{
+    [Fact] // the CI system redirects stdout, so we can only really test the redirected behavior.
```

Comment:
I'm not clear on why the CI system affects our ability to test it not being redirected.  Can't we schedule a process that sends the result back through some other mechanism, e.g. writing the result to a pipe that we then read and verify here?